required/latex-lab/latex-lab-sec.dtx

```diff
@@ -651,6 +662,7 @@
        \MakeLinkTarget#1{#2}
      }
   }
+\AddToNoCaseChangeList{\@hyp@section@target@n}  
```

Comment:
Just a thought: I don't know how long this no case change list already is, but wouldn't it be faster (and perhaps even better readable, if there is a general purpose \donotchangecase{...} with one argument that is used around material that should not change case?

Reply:
Do you mean `\NoCaseChange`?

Reply:
I could instead put everywhere \NoCaseChange around the command if that is is faster, not sure.

Reply:
it probably is faster as it has no effect on MakeUppercase in general whereas adding to the list means that every use of MakeUppercase has to check for `\@hyp@section@target@n`

Reply:
If we need more speed, I an always rework how `\AddToNoCaseChangeList` works

Reply:
Yes I meant that, I wasn't sure that surived in the final implementation. My guess is that it would be faster and and also clearer  codewise. what's your option @josephwright ?

Reply:
> If we need more speed, I an always rework how `\AddToNoCaseChangeList` works

maybe, but I don't see that it helps to have a long list if the task is always identical and understandably covered by a single command

Reply:
I switched to NoCaseChange